accounting/entries.go

```diff
@@ -231,15 +231,60 @@ func sweepEntries(tx lndclient.Transaction, u entryUtils) ([]*HarmonyEntry, erro
 	return []*HarmonyEntry{txEntry, feeEntry}, nil
 }
 
+// isUtxoManagementTx checks whether a transaction is restructuring our utxos.
+func isUtxoManagementTx(txn lndclient.Transaction) bool {
```

Comment:
I think this probably works well for the intended purpose. Though I wonder, how did such a transaction show up in `faraday` before? Was it just ignored or booked under another category? I'm not super familiar with all the functionality of `faraday` unfortunately...

Reply:
It was totally omitted, resulting in under-reporting on fees by faraday.